cmd/explaintest/r/explain_easy.result

```diff
@@ -357,13 +357,13 @@ Projection_11	5.00	root	9_aux_0
   ├─TableReader_15	5.00	root	data:TableScan_14
   │ └─TableScan_14	5.00	cop	table:t, range:[-inf,+inf], keep order:false
   └─StreamAgg_20	1.00	root	funcs:count(1)
-    └─IndexJoin_50	2.40	root	inner join, inner:TableReader_49, outer key:test.s.a, inner key:test.t1.a
-      ├─IndexReader_41	2.40	root	index:Selection_40
-      │ └─Selection_40	2.40	cop	eq(3, test.t.a)
-      │   └─IndexScan_39	3.00	cop	table:s, index:b, range:[3,3], keep order:false
-      └─TableReader_49	0.80	root	data:Selection_48
-        └─Selection_48	0.80	cop	eq(3, test.t.a)
-          └─TableScan_47	1.00	cop	table:t1, range: decided by [test.s.a], keep order:false
+    └─MergeJoin_45	2.40	root	inner join, left key:test.s.a, right key:test.t1.a
```

Comment:
Why is it changed?

Reply:
Previously, merge join is filtered out since planner think there should be a sort operator. After the fix, planner found that it don't need extra sort operator and the child table of the join is rather small. So it chose merge join.